### DISCUSSION TITLE
Long CDR3 sequences

### DISCUSSION THREAD

**Discussion by Januaryyiyue** [milaboratory/mixcr#1726] [Q&A] [answered]
I am using the `analyze` command with MiXCR version 4.6. I got the following CDR3 alpha clone: 
`CCLKDTFLPDTMYWGSSKVFRNDACGSALTAQLFF`
This seems to be an artefact. How can I remove such clones (e.g., clones longer than 20aas) from my analysis? Especially from the .clns file, because I am also using the `.clns` file for `postanalysis`.

Thank you so much.

**Comment by mizraelson** [comment, marked as answer]
Hi,
If you can export a list of clone IDs (excluding the IDs of the clones you want to filter out, for example, based on the length of the aaSeqCDR3) in a TSV file as a column of numbers, you can then run:

```
mixcr slice \
    --ids-file ids.tsv \
    input.clns \
    filtered.clns
```

`filtered.clns` will only contain the clones specified in the `ids.tsv` file.

**Reply by Januaryyiyue** [reply]
Thank you so much!
